### PR TITLE
i43–create new page called dashboard

### PR DESCRIPTION
**Pull request by eshwaribhide**
–Created new Dashboard.html page in page-content, same form/dropdown thing as Reports with 2 options for status dashboard/upcoming deadlines. Got rid of those options in Reports
–Created server-side Dashboard.js file, simply taking existing functionality for status dashboard/upcoming deadlines in Reports and moving it there
–Created client-side Dashboard functionality

### CODE REVIEW

**Comment by eshwaribhide** [comment]
So one thing is, I used the words "feature type" instead of "dashboard type" because I don't think that dashboard type makes sense as a reference to "upcoming deadlines". I understood that status dashboard & upcoming deadlines were two features, that is why I use feature/feature type. Please suggest a different name, if you think that one isn't the best.

File: client-code/JavaScript.html [hunk 1]
```diff
@@ -41,6 +41,7 @@
         runnerHTML.withUserObject($("#gantt-content")).getPlaceholderHTML("* Coming Soon * <br> Enter WBS # Above");
         runnerHTML.withUserObject($("#change-content")).getPlaceholderHTML("Choose Change Type Above");
         runnerHTML.withUserObject($("#report-content")).getPlaceholderHTML("* Coming Soon * <br> Choose Report Type Above");
+        runnerHTML.withUserObject($("#dashboard-content")).getPlaceholderHTML("* Coming Soon * <br> Choose Dashboard Type Above");
```
Comment (kevinyu328) [inline client-code/JavaScript.html:44]:
```suggestion
        runnerHTML.withUserObject($("#dashboard-content")).getPlaceholderHTML("Choose Dashboard Type Above");

```

File: page-content/Index.html [hunk 2]
```diff
@@ -77,6 +81,12 @@ <h2>NER Project Management</h2>
         </div>
       </div>
 
+      <div class="tab-pane fade" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab">
+        <div class="container-lg page" id="dashboard">
```
Comment (kevinyu328) [inline page-content/Index.html:85]:
```suggestion
        <div class="container-lg page" id="page-dashboard">

```

File: server-code/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
+/*
+Document: JS code specific for status dashboard/upcoming deadlines
+*/
+
+/**
+ * Returns HTML content for the specified dashboard type
+ * @param {String} dashboardType 
+ * @return {String}
```
Comment (kevinyu328) [inline server-code/Dashboard.js:8]:
```suggestion
 * @param {String} dashboardType - The type of dashboard to be displayed.
 * @return {String} HTML formatted content for the selected dashboard type.
```

**Review by kevinyu328 — CHANGES_REQUESTED** [review]
Really nice work! Just some small edits

File: client-code/JavaScript.html [hunk 4]
```diff
@@ -170,4 +171,27 @@
         getReportForm();
     })
 
+    /* Dashboard page functions */
+
+    /*
+    * Gets the appropriate dashboard type according to the type selected in dashboard_type dropdown and places it inside the dashboard_content wrapper
+    *
```
Comment (kevinyu328) [inline client-code/JavaScript.html:178]:
```suggestion
    * 
    * @return {void}

```

**Review by kevinyu328 — APPROVED** [review]
Looks good!

**Pull request merged**